docs/rules/generator-star-spacing.md

```diff
@@ -85,7 +104,7 @@ Examples of **correct** code for this rule with the `"before"` option:
 /*eslint generator-star-spacing: ["error", {"before": true, "after": false}]*/
 /*eslint-env es6*/
 
-function *generator() {}
+function *generator() {};
```

Comment:
True enough.. I added them for consistency, though I realize now that these didn't have them before because they're function declarations.